benchmarks/bench_filter_stamps.py

```diff
@@ -30,7 +39,7 @@ def setup_coadd_stamp(params):
     p = PSF(1.0)
     psf_dim = p.get_dim()
     psf_rad = p.get_radius()
-    for i in range(psf_dim):
+    for i in range(1, psf_dim):
```

Comment:
I'm not sure why this is changing. The calculation below moves the center pixel one off, but shouldn't the convolution with the PSF still cover the full PSF?

Reply:
before the range of the outer for loop would include -1 when i is 0 (due to the one pixel offset), giving an out of bound error. I just changed the range to only include dimensions that fall within the PSF.